sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -302,7 +303,30 @@ def test_lasso_cv_positive_constraint():
     assert min(clf_constrained.coef_) >= 0
 
 
-# FIXME: 'normalize' to be removed in 1.2
+def _scale_alpha_inplace(estimator, n_samples):
```

Comment:
I think we should keep this helper function definition just before the single test that uses it, namely `test_linear_model_sample_weights_normalize_in_pipeline` to avoid having to scroll to much to understand what's the test is doing.

Reply:
Actually there are now 2 tests that use it... Maybe we should group those 2 tests and put `_scale_alpha_inplace` just before.

Reply:
I just realized this is already the case. Please ignore my comments...